### DISCUSSION TITLE
Commit view Next button at bottom

### DISCUSSION THREAD

**Discussion by manish-hashicorp** [community/community#69719] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

In our team, we tend to have PRs with self-contained commits where one reviews commit by commit and clicks on "Next" button at the top of the commit page in a PR. It will be great if that button can be at the bottom of the page since after reviewing the commit I am always at the bottom and want to go to next commit.

**Comment by Sazidul0** [comment]
- Use keyboard shortcuts (e.g., Cmd/Ctrl + → for next commit).

- Install browser extensions like GitHub Next Commit at Bottom (custom scripts) to force button placement.

- Advocate for a team convention to collapse file diffs after review, reducing vertical scrolling.

**Internal Process Adjustment**
If this is specific to the Cypress repository:

- Check if the team maintains custom GitHub UI scripts or CSS overrides.

- Propose a PR to modify the navigation button placement using GitHub’s DOM manipulation APIs.
